cdec/installation-manager-core/src/test/java/com/codenvy/im/managers/TestConfigManager.java

```diff
@@ -131,7 +133,8 @@ public void testLoadDefaultMultiServerCdecConfig() throws Exception {
                                              "user_ldap_dn=dc=codenvy-enterprise,dc=com\n" +
                                              "billing_resources_refill_cron=0 15 0 * * ?\n" +
                                              "admin_ldap_dn=dc=codenvycorp,dc=com\n");
-        doReturn(properties).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class));
+        doReturn(properties).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class),
+                                                      MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
Leads to _Mockito_ error `Unfinished stubbing detected`.
Fixup: `eq(MediaType.APPLICATION_OCTET_STREAM)`